moja-kuhinjica-frontend/src/pages/editProfile/[id].tsx

```diff
@@ -44,9 +61,30 @@ const EditProfilePage = (): JSX.Element => {
         }
     }, [windowWidth])
 
+    const fetchUser = (): void => {
+        const res = UserService.getUserById(id)
```

Comment:
Same thing remove this `res` cause it is not used anywhere

Reply:
Removed.